inst/resources/vendor/renv.R

```diff
@@ -27,6 +21,13 @@ renv$initialize <- function() {
   script <- system.file("vendor/renv.R", package = .packageName)
   sys.source(script, envir = renv)
 
+  # set up metadata environment
+  renv$the$metadata <- as.environment(..metadata..)
+
+  # run our load / attach hooks so internal state is initialized
+  renv$renv_zzz_load()
+  renv$renv_zzz_attach()
```

Comment:
I think we should consider not running `renv_zzz_attach()` here, since renv is _not_ attached (and it looks like the code currently run there is not relevant to the embedded scenario).

Reply:
I think you're right.